src/generators/api-links/index.mjs

```diff
@@ -54,9 +50,9 @@ export default {
     if (input.length > 0) {
       const repositoryDirectory = dirname(input[0].path);
 
-      const repository = getBaseGitHubUrl(repositoryDirectory);
+      const repository = 'https://github.com/nodejs/node';
 
-      const tag = getCurrentGitHash(repositoryDirectory);
+      const tag = 'HEAD';
```

Comment:
Is it fine for us to assume HEAD?

Reply:
In dev it shouldn't really matter, but for releases it needs to be a hash since the line numbers referenced are going to change eventually.

I.e. for the latest v20 release, https://github.com/nodejs/node/blob/bb1a61d8737feff534bb85368dab3b7c554c863d/lib/_http_client.js#L869 is correct vs https://github.com/nodejs/node/blob/HEAD/lib/_http_client.js#L869 points to something else.